include/riak_kv_ts.hrl

```diff
@@ -71,8 +71,8 @@
           'SELECT'              :: #riak_sel_clause_v1{},
           'FROM'        = <<>>  :: binary() | {list, [binary()]} | {regex, list()},
           'WHERE'       = []    :: [riak_ql_ddl:filter()],
-          'ORDER BY'    = []    :: [riak_kv_qry_compiler:sorter()],
-          'LIMIT'       = []    :: [riak_kv_qry_compiler:limit()],
+          'ORDER BY'            :: undefined | [riak_kv_qry_compiler:sorter()],
+          'LIMIT'               :: undefined | riak_kv_qry_compiler:limit(),
```

Comment:
We shouldn't change old records, because they still don't change on the old versioned nodes that create them.